packages/front-end/components/Experiment/SinglePage.tsx

```diff
@@ -194,6 +195,11 @@ export default function SinglePage({
   editPhases,
   editPhase,
 }: Props) {
+  const [hideNewExperimentHelp, setHideNewExperimentHelp] = useLocalStorage(
+    `experiment-page__hide-new-experiment-help`,
+    false
```

Comment:
Should we be smarter about the default value?

```ts
let alreadyUsingNewPage = false;
try {
  if (window.localStorage.getItem("single-page-new-ui-v1") === "true") {
    alreadyUsingNewPage = true;
  }
} catch(e) {
  // Ignore localStorage errors
}

const [hideNewExperimentHelp, setHideNewExperimentHelp] = useLocalStorage(
    `experiment-page__hide-new-experiment-help`,
    // Hide if the user has already been using the new page
    alreadyUsingNewPage
);
```

Reply:
incorporated your suggestion